src/main/java/com/eppo/sdk/helpers/FetchConfigurationsTask.java

```diff
@@ -7,38 +7,31 @@
 
 public class FetchConfigurationsTask extends TimerTask {
   private static final Logger log = LoggerFactory.getLogger(FetchConfigurationsTask.class);
-  private final ConfigurationStore configurationStore;
+  private final Runnable runnable;
```

Comment:
By passing in a runable we can pass in a lambda which is nice for, among other things, having this leverage a protected `BaseEppoClient` method.